tests/unit/RestfulServerTest.php

```diff
@@ -142,9 +146,14 @@ public function testAuthenticatedPUT()
         $this->assertEquals(401, $response->getStatusCode()); // Permission failure
 
         $_SERVER['PHP_AUTH_USER'] = 'editor@test.com';
+
+        $response = Director::test($url, $data, null, 'PUT');
```

Comment:
We shouldn't shoe-horn a fairly fundamental authentication test into another unrelated test.

Please can you create a new test function like `testApiAccess` which adds additional authenticators and then asserts that the second authenticator works even if the first fails?